src/layouts/Home/components/Foundation.tsx

```diff
@@ -83,8 +83,8 @@ export const Foundation: React.FC = () => {
   const { contentImage, componentsImage, designImage } = useStaticQuery(
     graphql`
       {
-        contentImage: abstractAsset(layerName: { eq: "home-pillars-content" }) {
-          childFile {
+        contentImage: figmaAsset(name: { eq: "home-pillars-content" }) {
```

Comment:
There is a right-left jitter on the progressive load of this image that can be seen on a hard page refresh. Let's see if we can get this right.

Reply:
I'm also seeing the jitter. Just to be sure, can someone send me the source Figma file so I can see if there's a problem with the way the artboard is set up there?

Reply:
The assets are being pulled from Figma file `HifWgSrdeTlMMXUrcnrAAH`.

I only notice the jitter on the "home-pillars-content" image, but not the other two images. I replaced the avatar "Do" image locally, and it also doesn't seem to have a jitter. So maybe the jitter only affects the "home-pillars-content"?

@bethanybeech @james-sann, does it look like there's a problem with that particular asset? Maybe we can re-create that asset and I can try that instead?

Reply:
@hzhu Re-created the `home-pillars-content` frame. Let me know how you go 🤞

I placed the older version to the side of it in the Figma file and named it `home-pillars-content-old`

Just a side-note on these images and the website conversion. It appears we are surfacing raster images and not SVGs? (at least, that's what I see in Chrome and Safari) Is that correct? When I zoom into those images on the website, I get pixelations but typically SVG doesn't do that? Or does our plugin only support raster images? 
![image](https://user-images.githubusercontent.com/85654216/127818835-d88291ff-79eb-448a-ac67-94d36659bb71.png)

Reply:
If we can't get these to (at a minimum) render as well as what we currently see on production, it might make sense to add and reference SVGs directly from the `/website/static` folder.

Reply:
Thanks @james-sann for re-creating the frame. 

Unfortunately, the image still has a jitter. It seems like the placeholder image is currently generated with the option of "TRACED_SVG", and maybe there is an inaccuracy with the tracing. Using the default option "DOMINANT_COLOR" addresses the jitter, but removes the tracing feature. Would that be acceptable? Here's a [demo of that](https://ecstatic-benz-474ab9.netlify.app/).

Alternatively, I can put a SVG into the `/website/static` folder for `home-pillars-content`.

Reply:
The `home-pillar-*` images are now powered by static SVGs (exported from Figma). 

Since the PR aims to test Figma API integration, I've decided to migrate at least one image that sources from Figma directly, so this PR now sources the `home-hero-logo`. It looks like there are some issues on the Figma side. There's an extra white background, and it seems like a layer might be missing. This can be visible on the latest "View deployment" link.

@james-sann would you be able to check out what might be wrong with `home-hero-logo`?

Reply:
Hi @hzhu. Try it now 🤞 

I removed these settings from Figma and it should remove any background color fill getting exported (you can see the preview within Figma's export panel).

![image](https://user-images.githubusercontent.com/85654216/128135016-821a147e-1d93-4a1a-91a1-36780df9040b.png)

Reply:
@james-sann, that seemed to have worked, thanks!

Reply:
@james-sann oops, looks like the home hero image successfully removed the white background, but some layers seem to be missing.